test/access/extensions/AccessControlDefaultAdminRules.test.js

```diff
@@ -16,8 +16,7 @@ contract('AccessControlDefaultAdminRules', function (accounts) {
   it('initial admin not zero', async function () {
     await expectRevert(
       AccessControlDefaultAdminRules.new(delay, constants.ZERO_ADDRESS),
-      'AccessControlInvalidDefaultAdmin',
-      [constants.ZERO_ADDRESS],
+      "The transaction receipt didn't contain a contract address."
```

Comment:
I am not sure if this is the way to do it. Since not having a contract address on the receipt is our issue right? It should be some other revert reason.

Reply:
I've added a new file where we have a new `expectRevert` function that lets you decide the check strategy: in this specific case we just check the tx gets reverted